Impossible-Agent-Suite/SwapFees/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "impossible-6",
```

Comment:
Is the convention for agent names to be capitalized or lowercase?
`IMPOSSIBLE-6` or `impossible-6`?

Reply:
We are not being to strict with the name, but the name should be full lowercase or you will get this error 
> String does not match the pattern of "^(?:@[a-z0-9-*~][a-z0-9-*._\~]*/)?[a-z0-9-~][a-z0-9-._~]*$".

And we are not being to strict because the agent name is not used when an agent is deployed via Forta Connect